README.md

```diff
@@ -28,7 +28,7 @@ This milestone consists of two (2) sections:
 
 ## Live Demo
 
-[Live Demo Link](https://cranky-haibt-bf176a.netlify.app/)
+[Live Demo Link](https://stupefied-payne-1fc0b5.netlify.app/)
```

Comment:
*Changes Requested*
- [ ] Well done deploying a live version of the project to netlify. However, it is required that the project is deployed rather using Github Pages. Let us make these changes and move forward.

![deploy](https://user-images.githubusercontent.com/7220167/130429356-d15ee6d2-2dda-48e6-aaf5-567a3e4de314.png)

Reply:
Hello @oxenprogrammer, I deployed using github pages initially but the live link generated has refused to load the images I have on my project. Is there a way forward to this?

Reply:
I have fixed it up. It now works fine.